R/calculate_stay.R

```diff
@@ -38,5 +51,8 @@ calculate_stay <- function(year, start_date, end_date, sc_qtr = NULL) {
       # if end date is missing set to end of quarter
       lubridate::time_length(lubridate::interval(start_date, qtr_end), unit = "days")
     )
+  } else {
+    # Normal calculation if sc_qtr is supplied and end_date is not missing
+    lubridate::time_length(lubridate::interval(start_date, end_date), unit = "days")
```

Comment:
This bit of code will never be executed since the above if / if else are binary i.e. one of them will always be executed, either `sc_qtr` is missing or it's not.